### ISSUE TITLE
Can you please upload the pretrained model.

### ISSUE THREAD

**Issue by DRAhmadFaraz** [leVirve/lsun-room#7]
Can you please upload the pretrained model.? I ll be thankful to you.

**Comment by SpencerKD** [comment]
Can you please upload the pretrained model.? I ll be very very very thankful to you. Or can you please upload the LSUN dataSet including the labels of GT for the semantic segmentation

**Comment by DRAhmadFaraz** [comment]
I dont have pre-trained model

**Comment by leVirve** [comment]
Update at readme 😄 https://github.com/leVirve/lsun-room/tree/master#information

**Comment by sk-** [comment]
@leVirve Could you make the pretrained model public. Whenever I try to access the file it says I don't have permissions and I need to request access.
